src/main/PlagueMain.java

```diff
@@ -1078,7 +1081,7 @@ void loadMap(String args[]) {
         Log.info("Loading map " + map.name());
 
         world.loadMap(map);
-        loadedMap = state.map;
+        mindustry.maps.Map loadedMap = state.map;
```

Comment:
ew use the small signature

Reply:
I don't understand what you're asking me to do here. Can you elaborate?

Reply:
`Map loadedMap = state.map`
Or, `String mapID = state.map.file.name();` because thats the only place its ever used

Reply:
Map is already a reserved token in Java. It has to stay this way.

Reply:
Then do the other option, or remove the type, your not increasing readability.

Reply:
done